app/routes/project/posts/new.js

```diff
@@ -20,7 +20,7 @@ export default Ember.Route.extend({
 
   actions: {
     viewPost(post) {
-      this.transitionTo('project.posts.post', post);
+      this.transitionTo('project.posts.post', post.get('number'));
```

Comment:
The `project.posts.post` route uses a complex set of models which include the specified post, it's comments and a new, unsaved comment, so we need to pass in the post number instead of an already retrieved post model.

The number is all we need, since we already have the slugged route and project slugs from the parent route params.